server_config.txt

```diff
@@ -8,8 +8,8 @@ dh /etc/openvpn/easy-rsa/pki/dh2048.pem
 topology subnet
 server 10.8.0.0 255.255.255.0
 # Set your primary domain name server address for clients
-push "dhcp-option DNS 8.8.8.8"
-push "dhcp-option DNS 8.8.4.4"
+push "dhcp-option DNS 9.9.9.9"
+push "dhcp-option DNS 149.112.112.112"
```

Comment:
Same as [here](https://github.com/pivpn/pivpn/pull/891/files#r362763211). These IPs will be replaced upon installation.

Reply:
I liked the original approach of using "actual" resolvers as a just-in-case; I just thought I'd change the just-in-case to the suggested default.  Best to leave as-is so as not to cause confusion from those reviewing the script before executing?

Reply:
Good point.